.github/workflows/test.yaml

```diff
@@ -52,12 +54,13 @@ jobs:
       - uses: actions/checkout@v4
       - uses: foundry-rs/foundry-toolchain@v1
       - uses: ApeWorX/github-action@v3
```

Comment:
question: wait, what is this?

Reply:
we have a github action!
https://github.com/ApeWorX/github-action

Reply:
This is cool